src/services/user-config-service.ts

```diff
@@ -139,7 +153,7 @@ const convertYamlToUserConfig = (input?: string): Config => {
 
 const updateRepoConfigFromGitHub = async (repoSyncState: RepoSyncState, githubInstallationId: InstallationId, jiraHost: string, gitHubAppId: number | undefined): Promise<void> => {
 	const yamlConfig = await getRepoConfigFromGitHub(githubInstallationId, repoSyncState.repoOwner, repoSyncState.repoName, jiraHost, gitHubAppId);
-	const config = convertYamlToUserConfig(yamlConfig);
+	const config = await convertYamlToUserConfig(yamlConfig);
```

Comment:
Hmmm... was it broken before?

Reply:
No because of the FF I needed to make the function async that's why I needed to make it await here.